tools/debian/rules

```diff
@@ -11,7 +11,31 @@ endif
 %:
 	dh $@ --with=systemd
 
-override_dh_auto_configure:
+# Apply patches using git in order to support binary patches. Note that
```

Comment:
Is it worth noting here that we want to keep this part in sync with the spec file? https://github.com/cockpit-project/cockpit/blob/master/tools/cockpit.spec#L127

It might even be worth noting it in both places so that we don't forget to patch the other when we change one.